Core/fft_mt_r2iq.cpp

```diff
@@ -156,6 +156,7 @@ void fft_mt_r2iq::Init(float gain, ringbuffer<int16_t> *input, ringbuffer<float>
 
 		//        DbgPrintf((char *) "RandTable generated\n");
 
+#define HTLEN (halfFft / 4 + 1)
```

Comment:
shall we move this to config.h?

Reply:
It's a tentative code. It seems to me that length higher than (halfFft / 4 + 1) generates distortion may be a limit of the overlap and discharge implementation or some of my bugs...
Yes. The right value  is to use the max length possible and place the definition  in config.h
Some more tests are required. 
I'm not sure that halfFft /2 + 1 is usable without distortion... :-(